birdhouse/components/canarie-api/docker_configuration.py.template

```diff
@@ -109,10 +109,10 @@ SERVICES = {
             # NOTE:
             #   Below version and release time auto-managed by 'make VERSION=x.y.z bump'.
             #   Do NOT modify it manually. See 'Tagging policy' in 'birdhouse/README.rst'.
-            'version': '2.0.2',
-            'releaseTime': '2023-12-15T17:42:58Z',
-            'institution': '${SERVER_INSTITUTION}',
-            'researchSubject': '${SERVER_SUBJECT}',
+            'version': '2.0.4',
+            'releaseTime': '2024-01-18T14:27:21Z',
+            'institution': 'Ouranos',
```

Comment:
We should revert this change to this line and the one below it

Reply:
Right !  Instantiated value should not be inside a `.template` file.

Reply:
Patched with expected variables from upstream master.